### PR TITLE
[ doc ] update installation instructions to 1.7.1

### PR DESCRIPTION
**Pull request by laMudri**
I noticed that the installation instructions still refer to version 1.7, so I updated the versions mentioned to what I think the current ones are.

### CODE REVIEW

File: notes/installation-guide.md [hunk 1]
```diff
@@ -3,19 +3,19 @@ Installation instructions
 
 Note: the full story on installing Agda libraries can be found at [readthedocs](http://agda.readthedocs.io/en/latest/tools/package-system.html).
 
-Use version v1.7 of the standard library with Agda 2.6.2.
+Use version v1.7.1 of the standard library with Agda 2.6.2.1 or 2.6.2.
```
Comment (andreasabel) [inline notes/installation-guide.md:6]: While you are at it, the lastest release of Agda is 2.6.2.2.

**Review by andreasabel — CHANGES_REQUESTED** [review]
I think switching the docs to v1.7.1 is good, even though the only difference (afaik) is that v1.7.1 builds with GHC 9.2 while v1.7 doesn't.

Change request: if you mention 2.6.2.1, please also mention 2.6.2.2.

File: notes/installation-guide.md (hunk 1 above)
Comment (laMudri) [reply notes/installation-guide.md]: Does https://wiki.portal.chalmers.se/agda/pmwiki.php?n=Libraries.StandardLibrary also need to be updated to mention 2.6.2.2?

Comment (andreasabel) [reply notes/installation-guide.md]: Indeed.  Just did that.

**Review by andreasabel — APPROVED** [review]

**Comment by MatthewDaggitt** [comment]
Thanks!

**Pull request merged**